atests/res_setup.robot

```diff
@@ -9,28 +9,33 @@ ${HTTP_LOCAL_SERVER}    http://localhost:5010
 
 
 *** Keywords ***
-
 Setup Test Session
-    ${test_session}=  Set Variable  test_session
-    Set Test Variable  ${test_session}
-    Create Session  ${test_session}  ${HTTP_LOCAL_SERVER}
+    ${test_session}=    Set Variable    test_session
+    Set Test Variable    ${test_session}
+    Create Session    ${test_session}    ${HTTP_LOCAL_SERVER}
 
 Teardown Test Session
     Delete All Sessions
+    # Restore global session
+    Create Session    ${GLOBAL_SESSION}    ${HTTP_LOCAL_SERVER}
```

Comment:
The test setup deletes all sessions. So the teardown should make sure the global session exists for the remaining tests

Reply:
I'm not sure because the idea of the global session is to be reused.
In this way is deleted and created for each test.
Maybe we should implement a delete session single. By the way probably we could just ignore this and go ahead.

Reply:
I agree, a keyword to delete a specific session would be nice. Then acceptance tests can be much cleaner. I can work on this. Do you want it as part of this PR? Might become a bit messy..

Reply:
No no I think this PR is just fine. And I'm not sure when the single delete session might be useful for a real user but maybe exactly in a situation like ours.